### PR TITLE
VZ-7661 'psrctl explain' unit tests

### PR DESCRIPTION
**Pull request by stevshap**
Adding unit tests for the 'psrctl explain' command

### CODE REVIEW

File: tools/psr/psrctl/cmd/explain/explain_test.go [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+const psrRoot = "../../.."
+
+var (
+	expectedId          = "ops-s1"
+	expectedName        = "opensearch-s1"
+	expectedDescription = "This is a scenario that writes logs to STDOUT and gets logs from OpenSearch at a moderated rate."
+	expectedUseCase     = "Usecase path opensearch/writelogs.yaml:  Description: write logs to STDOUT 10 times a second"
+
+	// ConfigMap for the ops-s1 scenario
+	cm = &corev1.ConfigMap{
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/explain/explain_test.go:35]: you don't need this

File: tools/psr/psrctl/cmd/explain/explain_test.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+		WorkerChartAbsDir: psrRoot + "/manifests/charts/worker",
+		UseCasesAbsDir:    psrRoot + "/manifests/usecases",
+		ScenarioAbsDir:    psrRoot + "/manifests/scenarios",
+	}
+
+	defer manifest.ResetManifests()
+
+	defer func() { k8sutil.GetCoreV1Func = k8sutil.GetCoreV1Client }()
+	k8sutil.GetCoreV1Func = func(log ...vzlog.VerrazzanoLogger) (corev1cli.CoreV1Interface, error) {
+		return k8sfake.NewSimpleClientset(cm).CoreV1(), nil
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/explain/explain_test.go:83]: you don't need to add the configmap to the clientset

File: tools/psr/psrctl/cmd/explain/explain.go [hunk 3]
```diff
@@ -34,7 +34,7 @@ func NewCmdExplain(vzHelper helpers.VZHelper) *cobra.Command {
 	cmd.Example = helpExample
 
 	cmd.PersistentFlags().StringVarP(&scenarioID, constants.FlagScenario, constants.FlagsScenarioShort, "", constants.FlagScenarioHelp)
-	cmd.PersistentFlags().BoolVarP(&verbose, constants.FlagVerbose, constants.FlagVerboseShort, true, constants.FlagVerboseHelp)
+	cmd.PersistentFlags().BoolVarP(&verbose, constants.FlagVerbose, constants.FlagVerboseShort, false, constants.FlagVerboseHelp)
```
[the remark this answers is not in the document]
Comment (stevshap) [reply tools/psr/psrctl/cmd/explain/explain.go:37]: I feel like verbose should be disabled as default? Feels weird to do `-v false` to disable it

[the remark this answers is not in the document]
Comment (mbenmclean) [reply tools/psr/psrctl/cmd/explain/explain.go:37]: ok that's fine

**Review by mbenmclean — APPROVED** [review]

**Pull request merged**